docs/puzzles/day21.md

```diff
@@ -1,11 +1,312 @@
 import Solver from "../../../../website/src/components/Solver.js"
 
-# Day 21
+# Day 21: Dirac Dice
+by @sjrd
 
 ## Puzzle description
 
 https://adventofcode.com/2021/day/21
 
+## Modeling and parsing the input
+
+There are no more surprises here, so let us directly see the code:
+
+```scala
+type Cell = Int // from 0 to 9, to simplify computations
+
+case class Player(cell: Cell, score: Long)
+
+type Players = (Player, Player)
+
+def parseInput(input: String): Players =
```

Comment:
I don't think `Players` is better than  `(Player, Player)` because it hides that there can only be 2 players.

```suggestion
def parseInput(input: String): (Player, Player) =
```

Reply:
sorry I should have reviewed that earlier. It is okay if you leave it this way.